core/shared/src/test/scala/fs2/CompressionSuite.scala

```diff
@@ -111,7 +111,7 @@ abstract class CompressionSuite(implicit compression: Compression[IO]) extends F
     }
   }
 
-  test("inflate input") {
+  test("inflate input".ignore) { // TODO: This test hangs after upgrading to latest scalacheck-effect release, which fixed generators to increase sizes
```

Comment:
@armanbilge I haven't looked in to why yet but this test hangs (and doesn't even timeout) when against the new scalacheck-effect. The only difference should be that the string parameter grows to be more than 1 character long now.

Reply:
Just on JS I assume? I remember having some problems in the inputs got too big but that doesn't seem to be the case here. I'll look into it.

Reply:
Wait I misread that. I saw this PR to scalacheck-effect. So actually the inputs were only 1-character long all along?